client_cli/src/main.rs

```diff
@@ -830,15 +819,12 @@ mod asset {
     /// Transfer asset between accounts
     #[derive(clap::Args, Debug)]
     pub struct Transfer {
-        /// Account from which to transfer (in form `name@domain_name')
-        #[arg(short, long)]
-        pub from: AccountId,
         /// Account to which to transfer (in form `name@domain_name')
         #[arg(short, long)]
         pub to: AccountId,
```

Comment:
I think `short` can be removed here as well

Reply:
I think it should also be removed for `AssetDefinitionId` in `Register`